Content.Shared/Damage/DamageTypePrototype.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.Manager.Attributes;
+
+namespace Content.Shared.Damage
+{
+    /// <summary>
+    ///
+    /// </summary>
```

Comment:
Empty summary.
```suggestion
```

Reply:
this is a valid summary for damage type. it encompasses the void that was before and after